benchmarks/backgrounds/Snakefile

```diff
@@ -0,0 +1,110 @@
+import os
+import shutil
+
+from snakemake.remote.S3 import RemoteProvider as S3RemoteProvider
+
+
+S3 = S3RemoteProvider(
+    endpoint_url="https://eics3.sdcc.bnl.gov:9000",
+    access_key_id=os.environ["S3_ACCESS_KEY"],
+    secret_access_key=os.environ["S3_SECRET_KEY"],
+)
+
+
+rule backgrounds_get_beam_gas_electron:
+    input:
+        S3.remote("eictest/EPIC/EVGEN/BACKGROUNDS/BEAMGAS/electron/beam_gas_ep_10GeV_foam_emin10keV_10Mevt_vtx.hepmc"),
+    output:
+        "input/backgrounds/beam_gas_electron.hepmc",
+    run:
+        shutil.move(input[0], output[0])
+
+
+rule backgrounds_get_beam_gas_proton:
+    input:
+        S3.remote("eictest/EPIC/EVGEN/BACKGROUNDS/BEAMGAS/proton/ProtonBeamGasEvents/100GeV/100GeV_1.hepmc"),
+    output:
+        "input/backgrounds/beam_gas_proton.hepmc",
+    run:
+        shutil.move(input[0], output[0])
+
+
+rule backgrounds_get_DIS:
+    input:
+        S3.remote("eictest/EPIC/EVGEN/DIS/NC/{BEAM}/minQ2={MINQ2}/pythia8NCDIS_{BEAM}_minQ2={MINQ2}_{SUFFIX}.hepmc"),
+    wildcard_constraints:
+        BEAM="\d+x\d+",
+        MINQ2="\d+",
+    output:
+        "input/backgrounds/pythia8NCDIS_{BEAM}_minQ2={MINQ2}_{SUFFIX}.hepmc",
+    run:
+        shutil.move(input[0], output[0])
+
+
+rule backgrounds_sim:
+    input:
+        "input/backgrounds/{NAME}.hepmc",
+    output:
+        "sim_output/{DETECTOR_CONFIG}/{NAME}.edm4hep.root",
+    log:
+        "sim_output/{DETECTOR_CONFIG}/{NAME}.edm4hep.root.log",
+    params:
+        N_EVENTS=100
+    shell:
+        """
+ddsim \
+  --runType batch \
+  --part.minimalKineticEnergy 100*GeV  \
+  --filter.tracker edep0 \
+  -v WARNING \
+  --numberOfEvents {params.N_EVENTS} \
+  --compactFile $DETECTOR_PATH/{wildcards.DETECTOR_CONFIG}.xml \
+  --inputFiles {input} \
+  --outputFile {output}
+"""
+
+
+rule backgrounds_org2py:
+    input:
+        notebook=workflow.source_path("ecal_backwards.org"),
```

Comment:
Is the `.org` a canonical extension? Not familiar with it. It's python, right? So no `.py`?

Reply:
For org-mode, yes.